documentation/docs/get-started/setup.mdx

```diff
@@ -14,37 +14,61 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ```
 
-## Setup
+## Setup Khoj
 These are the general setup instructions for self-hosted Khoj.
-You can install the Khoj server using either Docker or Pip.
+You can install the Khoj server using either [Docker](?method=docker) or [Pip](?method=pip).
```

Comment:
How does this deep-linking work? I don't see the `method` ID being used.

Reply:
Good catch, fixed by converting to use `server` which is a valid query param from Tabs